### DISCUSSION TITLE
github achievements collection

### DISCUSSION THREAD

**Discussion by vanesagab** [community/community#21043] [Other Feature Feedback, Questions, & Ideas] [answered]
What are all the available badges out there? And how to get them?

**Comment by Griphcode** [comment]
Here is all the available Badges you can get:

**Pair Extraordinaire** = [Coauthored](https://docs.github.com/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) in a merged pull request
**Quickdraw** = Closed an issue or a pull request within 5 min of opening
**Starstruck** = Created a repository that has 16 stars
**Galaxy Brain** = 2 accepted answers
**Pull Shark** = 2 pull requests merged
**YOLO** = Merged a pull request without code review
**Public Sponsor** = Sponsoring open source work via [GitHub Sponsors](https://github.com/sponsors)


Hope this helps

**Comment by melvincwng** [comment]
@vanesagab, check out this repository over here for a full list of all available achievements - https://github.com/Schweinepriester/github-profile-achievements

Hope this helps

**Comment by radar-blip** [comment]
@vanesagab, They are these listed here: https://github.com/drknzz/GitHub-Achievements

**Reply by siakamansaly** [reply]
thanks !

**Comment by manasesjesus** [comment, marked as answer]
There are different types of badges. 

**STAFF** is for employees of GitHub.
**PRO** is for students with the GitHub Student Developer Pack, teachers too, and paid accounts.
You can check all these types here: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#displaying-badges-on-your-profile

The Achievements stickers/badges were introduced recently and some will be more like easter eggs that you'll stumble upon later. You can read more about here: https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey

So far, the unlocked achievements are documented here: https://github.com/drknzz/GitHub-Achievements

**Comment by diobenu2silva** [comment]
I can exchange stars. Please help each other.

https://github.com/diobenu2silva/pumpfun_SC
https://github.com/diobenu2silva/zeroGpt4
https://github.com/diobenu2silva/solArmy_staking_SC
https://github.com/diobenu2silva/triangular_arbitrage_multi_exchange
